hcx-apis/src/main/java/org/swasth/hcx/controllers/BaseController.java

```diff
@@ -94,4 +97,22 @@ private void processAndSendEvent(String apiAction, String metadataTopic, Map<Str
             return new ResponseEntity<>(errorResponse(response, ErrorCodes.SERVER_ERROR, e), HttpStatus.INTERNAL_SERVER_ERROR);
         }
     }
+
+    public Map<String, Object> formatsRequestBody(Map<String, Object> requestBody) {
+        Map<String, Object> event = new HashMap<>();
+        try {
+            String str = (String) requestBody.get("payload");
+
+            String[] strArray = str.split("\\.");
+            event.put("protected", strArray[0].isEmpty() ? " " : strArray[0] );
+            event.put("encrypted_key", strArray[1].isEmpty() ? " " : strArray[1]);
+            event.put("aad", strArray[2].isEmpty() ? " " : strArray[2]);
+            event.put("iv", strArray[3].isEmpty() ? " " : strArray[3]);
+            event.put("ciphertext", strArray[4].isEmpty() ? " " : strArray[4]);
+            event.put("tag", strArray[5].isEmpty() ? " " : strArray[5]);
+        }catch (Exception e){
+
+        }
```

Comment:
We should not suppress the exception. We have to handle the exception and return the client error if there is any error with the request.

Reply:
Done